cilium/cmd/bpf_ct_flush.go

```diff
@@ -32,6 +32,7 @@ var bpfCtFlushCmd = &cobra.Command{
 	PreRun: requireEndpointIDorGlobal,
 	Run: func(cmd *cobra.Command, args []string) {
 		common.RequireRootPrivilege("cilium bpf ct flush")
+		initSupportedMapTypes()
```

Comment:
Not sure if we should cache it somehow. Usually times of `bpftool feature -j` on my machine look like:

```
real	0m0.147s
user	0m0.018s
sys	0m0.118s
```